### DISCUSSION TITLE
Tool to import directly from OneNote

### DISCUSSION THREAD

**Discussion by cprpolo** [zim-desktop-wiki/zim-desktop-wiki#2332] [Q&A]
Thank you for this app :)

I am trying to move away from One Note and I'd like to use Zim but I'd like to import the notes directly from the OneNote Exports, is this possible/feasible?

This may help:
https://developer.microsoft.com/en-US/onenote

But I have no clue about programming

Thank you in advance

**Comment by lorenrosen** [comment]
Looks like with a personal account you can export the notes in a zip file. (I don't know about the actual format of the notes.) So it might be just a matter of importing that into Zim, though I wouldn't be surprised if the details proved tricky to get right.

See
Export and import OneNote notebooks - Microsoft Support
 https://support.microsoft.com/en-us/office/export-and-import-onenote-notebooks-a4b60da5-8f33-464e-b1ba-b95ce540f309
